addons/zylann.hterrain/util/xyz_size_calc.gd

```diff
@@ -0,0 +1,37 @@
+class_name XYZMeta
+
+const MAX_INT = 9223372036854775807 # 2^63 - 1
+
+var size_x = -1
+var size_y = -1
+var min_x = MAX_INT
+var min_y = MAX_INT
+var max_x = 0
+var max_y = 0
+
+
+func get_XYZ_Meta(f, logger):
```

Comment:
Function names should be lowercase.

Also you made that script such that it has to be instanced with `new` before being used. This is not necessary. Usually utility functions like this one can be `static func`, member variables can be local, and the script can be `const preload` in the places it needs to be used.

You are also passing `logger` but you aren't using it.